ios/EnrichedTextInputView.mm

```diff
@@ -1355,16 +1355,16 @@ - (void)anyTextMayHaveBeenModified {
       }
     }
 
-    // emit string without zero width spaces
-    NSString *stringToBeEmitted = [[textView.textStorage.string
-        stringByReplacingOccurrencesOfString:@"\u200B"
-                                  withString:@""] copy];
-
     // emit onChangeText event
     auto emitter = [self getEventEmitter];
     if (emitter != nullptr) {
       // set the recently emitted string only if the emitter is defined
-      _recentlyEmittedString = stringToBeEmitted;
+      _recentlyEmittedString = [textView.textStorage.string copy];
```

Comment:
`_recentlyEmittedString` now kinda doesn't follow its name because we emit without zero width spaces but put a ZWS-containing string into it. I'd rename it to something like `_recentInputString`